tools/build_defs/cc/cc_import.bzl

```diff
@@ -71,6 +75,60 @@ def _perform_error_checks(
         not _is_shared_library_extension_valid(shared_library_artifact.basename)):
         fail("'shared_library' does not produce any cc_import shared_library files (expected .so, .dylib or .dll)")
 
+    if static_library != None and (bool(pic_object_files) or bool(nopic_object_files)):
+        fail("'static_library' and object files should not be provided at the same time")
+
+    if bool(pic_object_files) and bool(nopic_object_files):
+        fail("Only one type of object files (PIC or no-PIC) should be provided")
```

Comment:
Do these checks make sense?

Reply:
If yes, I should probably write tests for them...

Reply:
The first one makes sense. The second one doesn't, because it's up to the target later downstream to decide if it wants pic or no-pic, so we should make both available.